api/config/config.go

```diff
@@ -23,9 +47,49 @@ func init() {
 // DefaultConfig defines the default configuration options for api
 func DefaultConfig() Config {
 	return Config{
-		StartGrpcServer: defaultStartGRPCServer, // note: all bool flags default to false so don't set one of these to true here
-		GrpcServerPort:  defaultGRPCServerPort,
-		StartJSONServer: defaultStartJSONServer,
-		JSONServerPort:  defaultJSONServerPort,
+		StartGrpcServer:         defaultStartGRPCServer, // note: all bool flags default to false so don't set one of these to true here
+		StartGrpcServices:       nil,                    // note: cannot configure an array as a const
+		GrpcServerPort:          defaultGRPCServerPort,
+		NewGrpcServerPort:       defaultNewGRPCServerPort,
+		NewGrpcServerInterface:  defaultNewGRPCServerInterface,
+		StartJSONServer:         defaultStartJSONServer,
+		StartNewJSONServer:      defaultStartNewJSONServer,
+		JSONServerPort:          defaultJSONServerPort,
+		NewJSONServerPort:       defaultNewJSONServerPort,
+		StartNodeService:        defaultStartNodeService,
+		StartMeshService:        defaultStartMeshService,
+		StartGlobalStateService: defaultStartGlobalStateService,
+		StartTransactionService: defaultStartTransactionService,
+		StartSmesherService:     defaultStartSmesherService,
```

Comment:
what is the smesher service? perhaps choose a name actually describing the service

Reply:
It lets people who are smeshing configure their allocated space and coinbase, start and stop smeshing, etc. - can you think of a better name? 🤓

Reply:
Sinc ethis is done for reward or block creation - either rewardSettings or BlockCreationParam?

Reply:
I think the name is okay, let me know if you feel strongly otherwise